File: app/src/main/java/com/example/dzj/myreader/modle/Paragraph.java
package com.example.dzj.myreader.modle;

public class Paragraph {
	private String strParagraph;
	private int startPage;
	private int startLine;
	private int endPage;
	private int endLine;
	
	public Paragraph(String str) {
		this.strParagraph = str;
	}
	
	public String getStrParagraph() {
		return strParagraph;
	}
	public void setStrParagraph(String strParagraph) {
		this.strParagraph = strParagraph;
	}
	public int getStartPage() {
		return startPage;
	}
	public void setStartPage(int startPage) {
		this.startPage = startPage;
	}
	public int getStartLine() {
		return startLine;
	}
	public void setStartLine(int startLine) {
		this.startLine = startLine;
	}
	public int getEndPage() {
		return endPage;
	}
	public void setEndPage(int endPage) {
		this.endPage = endPage;
	}
	public int getEndLine() {
		return endLine;
	}
	public void setEndLine(int endLine) {
		this.endLine = endLine;
	}
	public String toString() {
		return strParagraph;
	}
}

File: app/src/main/java/com/example/dzj/myreader/modle/Paragraph.kt
package com.example.dzj.myreader.modle

class Paragraph(var strParagraph: String?) : Cloneable {
    var startPage: Int = 0
    var startLine: Int = 0
    var endPage: Int = 0
    var endLine: Int = 0
    override fun toString(): String {
        return strParagraph.toString()
    }

    public override fun clone(): Any {
        var paragraph: Paragraph? = null
        try {
            paragraph = super.clone() as Paragraph
        } catch (e: CloneNotSupportedException) {
            e.printStackTrace()
        }

        return paragraph!!
    }
}
